### PR TITLE
Display full img's description

### PR DESCRIPTION
**Pull request by Lyczeq**
**Description**
Displaying all features provided from lambdas and GCP of the uploaded photo

**Changes**
- Add own component for each result from GCP
- Display every result provided from GCP

**Related issue:** #91

### CODE REVIEW

File: frontend/src/views/ImageDetailsPage/components/TextTile/TextTile.styles.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+export const Template = styled.article`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 2%;
+  max-height: 260px;
+  overflow-x: hidden;
+  overflow-y: auto;
+`;
```
Comment (mwisnia97) [inline frontend/src/views/ImageDetailsPage/components/TextTile/TextTile.styles.js:11]: Maybe rename that styled component and move to global styles? I see you use two styles for tiles, so they can be reusable from global style. 🤔

File: frontend/src/views/ImageDetailsPage/components/LandmarksTile/LandmarksTile.styles.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  overflow-x: hidden;
+  overflow-y: auto;
+
+  li {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 4% 0;
+  }
+`;
```
Comment (mwisnia97) [inline frontend/src/views/ImageDetailsPage/components/LandmarksTile/LandmarksTile.styles.js:18]: FaceTile has the same style component, mby move. it to global styles 🤔

**Review by mwisnia97 — COMMENTED** [review]
My suggestion:
Move all tiles to folder tiles in ImageDetails/component for better file organization.

**Review by Wawrzyn321 — COMMENTED** [review]
LGTM

**Review by mwisnia97 — APPROVED** [review]

**Pull request merged**